### PR TITLE
[TW-1028] Doc offline API Client (Scratch Pad v2)

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Added new page: Getting Started > Using the Postman API Client
- Updated page: Getting Started > Using the Scratch Pad
- Updated other references to Scratch Pad

UPDATE: Renamed "offline API Client" to "lightweight API Client"

### CODE REVIEW

File: src/pages/docs/getting-started/using-api-client.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Moving from the API Client to a workspace
+
+[Workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/) enable you to organize your Postman work and collaborate with teammates. When you're in a workspace, you can save requests to [collections](/docs/collections/using-collections/), use [environments](/docs/sending-requests/managing-environments/) and [variables](/docs/sending-requests/variables/), build [APIs](/docs/designing-and-developing-your-api/the-api-workflow/), and more.
+
+To switch to a workspace, you need to sign up for a Postman account or sign in to your existing Postman account. You can move all of your open requests and history from the API Client to your workspace, so you can pick up right where you left off.
+
+### Signing up for Postman from the API Client
+
+If you don't have a Postman account, or want to use a new account, select **Create Account** in the header. Follow the onscreen instructions, or learn more about [signing up for Postman account](/docs/getting-started/postman-account/#signing-up-for-a-postman-account).
```
Comment (petewitcher-postman) [inline src/pages/docs/getting-started/using-api-client.md:57]:
```suggestion
If you don't have a Postman account, or want to use a new account, select **Create Account** in the header. Follow the onscreen instructions, or learn more about [signing up for a Postman account](/docs/getting-started/postman-account/#signing-up-for-a-postman-account).
```

**Review by petewitcher-postman — APPROVED** [review]
LGTM! I made one minor suggestion.

File: src/components/LeftNav/LeftNavItems.jsx [hunk 2]
```diff
@@ -93,6 +93,10 @@ export const leftNavItems = [
         name: 'About the Postman Agent',
         url: '/docs/getting-started/about-postman-agent/',
       },
+      {
+        name: 'Using the API Client',
```
Comment (giridharvc7) [inline src/components/LeftNav/LeftNavItems.jsx:97]: Calling this offline API Client would make even more sense here. 
Using just "API Client" can be confusing.

**Review by giridharvc7 — CHANGES_REQUESTED** [review]

File: src/components/LeftNav/LeftNavItems.jsx [hunk 3]
```diff
@@ -93,6 +93,10 @@ export const leftNavItems = [
         name: 'About the Postman Agent',
         url: '/docs/getting-started/about-postman-agent/',
       },
+      {
+        name: 'Using the offline API Client',
```
Comment (giridharvc7) [inline src/components/LeftNav/LeftNavItems.jsx:97]: Can we get a concensus on of the `O` in `Offline API Client` should be capitalized?

File: src/pages/docs/getting-started/using-offline-api-client.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+While you're in the offline API Client, you can create new requests, send requests and receive responses, and view the history of your sent requests.
+
+* To create a new request, select **New** in the sidebar and then select the type of request you want to create. You can create HTTP, WebSocket, gRPC, or GraphQL requests. Each request you create opens in a new tab. Learn more about [building requests](/docs/sending-requests/requests/).
+
+* Select **Send** to send a request and view the response. Learn more about [receiving responses](/docs/sending-requests/responses/).
+
+* The sidebar shows a history of the requests you've sent. Select a request in the history to open it in a new tab. Learn more about [viewing your request history](/docs/getting-started/navigating-postman/#history).
+
+> You can also use the offline API Client to write and execute [test scripts](/docs/writing-scripts/pre-request-scripts/) and to [visualize response data](/docs/sending-requests/visualizer/).
```
Comment (giridharvc7) [inline src/pages/docs/getting-started/using-offline-api-client.md:53]: Should we also mention about changing protocols of a request if needed?

**Review by giridharvc7 — CHANGES_REQUESTED** [review]

File: src/components/LeftNav/LeftNavItems.jsx (hunk 3 above)
Comment (jkonrath-postman) [reply src/components/LeftNav/LeftNavItems.jsx:97]: @giridharvc7 @mark-anthony-postman I just raised this question in Slack - let's find out there.

Comment (jkonrath-postman) [reply src/components/LeftNav/LeftNavItems.jsx:97]: Answer: It is lowercase o.

File: src/pages/docs/getting-started/using-offline-api-client.md (hunk 4 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/using-offline-api-client.md]: I added a note about using the protocol switcher

**Review by giridharvc7 — APPROVED** [review]

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**